src/index.html

```diff
@@ -1,13 +1,292 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <link
+      rel="shortcut icon"
+      href="./styles/images/svg/bike_link.png"
+      type="image/x-icon"
+    >
+    <title>MyBike</title>
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__header header">
+      <div class="header__bar">
+        <a href="#" class="header__logo">
+          <img
+            src="./styles/images/svg/MyBiKE.svg"
+            alt="header-logo-image"
+            class="header__logo-img"
+          >
+        </a>
+        <div class="header__links">
+          <a href="tel:+12345555555" class="header__phone">
+            <div class="header__phone-number">+1 234 555-55-55</div>
+            <img
+            src="./styles/images/svg/phone.svg"
+            alt="header-phone-icon"
+            class="header__phone-img">
+          </a>
+
+          <div class="header__menu-button">
+            <span class="header__menu-button-span"></span>
+            <span class="header__menu-button-span"></span>
+            <span class="header__menu-button-span"></span>
```

Comment:
Dont really understand why you added the icon using three spans for the white lines. You could simply add the full icon. Now your menu icon consists of three white lines and thats not what expected.

![image](https://user-images.githubusercontent.com/121725856/218118372-14692641-dc15-4a49-9779-f94a05575fa9.png)

Reply:
With span, you can smoothly resize on hover, and not substitute another image.